src/mixins/common-mixin.ts

```diff
@@ -61,10 +61,11 @@ export class CommonMixin extends Vue {
       case RequestCode.AML: return 'Amalgamation'
       case RequestCode.CHG: return 'Name Change'
       case RequestCode.CNV: return 'Alteration'
-      // case RequestCode.DBA:
-      // case RequestCode.ASSUMED:
-      // case RequestCode.REN:
-      // case RequestCode.REST:
+      case RequestCode.DBA: return 'Doing Business As'
```

Comment:
It's not clear if there will be used or are legacy codes, but Linda provided me wording for them so here they are.